pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
`options` is `nil` until the line further down, so this fails with `panic: runtime error: invalid memory address or nil pointer dereference` :smile: